### ISSUE TITLE
Haunted Kitchen

### ISSUE THREAD

**Issue by Bisten** [soolar/sl_ascend#130]
The script isn't maximizing stench and hot resist when doing the haunted kitchen. Not a major issue, but thought I would document it.

**Comment by MatthewDarling** [comment]
Looks how it's handled right now is:

  1. Check if you've toggled the `delayKitchen` setting
  2. If yes, don't go there unless you have 9+ resistance or you've reached `sl_powerLevelLastLevel`
  3. If no, apply these three skills/buffs: Hide of Sobek, Patent Prevention, Mist Form
  4. Then do the kitchen

So if those buffs aren't available, it will go ahead without resistance.

Is there a general list of "resources worth dedicating to resistance in Haunted Kitchen"? Like, is it worth spending a Latte refill, making a pull, wishing, etc. I think you only benefit if you can reach 3/6/9 in an element, so that's an extra wrinkle for small effects like random equipment.

**Comment by Bisten** [comment]
I don’t think it was casting mist form when I ran it. I also tend to pull the topiaria outfit because it puts you at 9 resist with mist form and is very useful in tons of places throughout the run. Probably not a pull everyone would want to do though.

**Comment by MatthewDarling** [comment]
I think this is still a valid issue - any reason why you've closed the issue? Seems like we should try to buff resistance with more than just three things, considering (if I've understood the wiki correctly) it takes up to 21 turns without any resistance.

**Comment by Bisten** [comment]
Sorry, I figured that casting mist form was enough. No resist is 21 turns, mist form is 10 turns, and the full topiaria outfit knocks it down to 6 turns. That’s 4 turns savings for 3 pulls, but the pulls are pretty universally helpful. It cuts turns from the hedge maze and a-boo peak. I’m not sure how the script handles pulls in this scenario.

**Comment by taltamir** [comment]
Low skill vamps might not have mist. Also mist form is only for vampyre, hide of sobek is for ed, and patent prevention is for west of loathing avatars.
If it only uses those 3 skills there is room for improvement.

I would suggest adding the following skills to it:
general - http://kol.coldfront.net/thekolwiki/index.php/Elemental_Saucesphere_(skill)
general - http://kol.coldfront.net/thekolwiki/index.php/Astral_Shell_(skill)
Avatar of jarlsberg - http://kol.coldfront.net/thekolwiki/index.php/Oilsphere_(effect)
nuclear autumn - http://kol.coldfront.net/thekolwiki/index.php/Flappy_Ears
turtle tamer - http://kol.coldfront.net/thekolwiki/index.php/Avatar_of_She-Who-Was

vampyre (only if mist form is unavailable as the two are incompatible and this is inferior) - http://kol.coldfront.net/thekolwiki/index.php/Spectral_Awareness_(effect)

and then do a maximizer call for whatever equipment is on hand. (such as may the fourth saber)
